webstruct/features/data_features.py

```diff
@@ -99,3 +101,52 @@ def looks_like_range(html_token):
     return {
         'looks_like_range': token in RANGES
     }
+
+
+def looks_like_ordinal(html_token):
+    if len(html_token.token) < 3:
+        return {'looks_like_ordinal': False}
+    if re.search('\d*1st', html_token.token):
+        return {'looks_like_ordinal': True}
+    if re.search('\d*2nd', html_token.token):
+        return {'looks_like_ordinal': True}
+    if re.search('\d*3rd', html_token.token):
+        return {'looks_like_ordinal': True}
+    if re.search('11th|12th|13th', html_token.token):
+        return {'looks_like_ordinal': True}
+    if re.search('\d*[4,5,6,7,8,9,0]th', html_token.token):
+        return {'looks_like_ordinal': True}
+    return {'looks_like_ordinal': False}
+
+
+def looks_like_date_pattern(html_token):
+    nchars = len(html_token.token)
+    if nchars < 8 or nchars > 10:
+        return {'looks_like_date_pattern': False}
+    if re.search('\d{1,2}\/\d{1,2}\/\d{2,4}', html_token.token):
+        return {'looks_like_date_pattern': True}  # XX/XX/XXXX
+    if re.search('\d{1,2}\.\d{1,2}\.\d{2,4}', html_token.token):
+        return {'looks_like_date_pattern': True}  # XX.XX.XXXX
+    if re.search('\d{1,2}-\d{1,2}-\d{2,4}', html_token.token):
```

Comment:
It matches XX.X.XXX, right? I think it makes sense to exclude 3-letter years from the pattern.

This function also doesn't catch common date variants like YYYY-MM-DD